geoips/yaml_configs/product_params/Uncorrected-Channel.yaml

```diff
@@ -19,5 +19,5 @@ Uncorrected-Channel:
         data_range: [0.0, 1.0]      # MUST pass data_range, it will not auto-detect
         cmap_name: Greys_r
         cbar_label: Normalized Channel Data
-    interp_func: pyresample_wrappers.interp_nearest
+    interp_func: interp_nearest
```

Comment:
@jsolbrig NOTE: I updated all the old product_params to use the new plugin names, but did NOT update the new YAML plugins since they are currently unused.  Making these udpates allows the existint test_full_install.sh to pass.

Reply:
Looks good, thank you!